src/Lean/Util/PtrSet.lean

```diff
@@ -21,7 +21,7 @@ unsafe instance : BEq (Ptr α) where
 /--
 Set of pointers. It is a low-level auxiliary datastructure used for traversing DAGs.
 -/
-unsafe def PtrSet (α : Type) :=
+unsafe abbrev PtrSet (α : Type) :=
```

Comment:
This seems unrelated?

Reply:
Oh, sorry. I did another modification that needed to write in a new PR.

Reply:
Now I fixed this. I moved the `PtrSet` modification to another branch.